pytorch3d/implicitron/dataset/load_blob.py

```diff
@@ -0,0 +1,545 @@
+                specified relative to this root (but not json paths themselves).
+        load_images: Enable loading the frame RGB data.
+        load_depths: Enable loading the frame depth maps.
+        load_depth_masks: Enable loading the frame depth map masks denoting the
+            depth values used for evaluation (the points consistent across views).
+        load_masks: Enable loading frame foreground masks.
+        load_point_clouds: Enable loading sequence-level point clouds.
+        max_points: Cap on the number of loaded points in the point cloud;
+                if reached, they are randomly sampled without replacement.
+        mask_images: Whether to mask the images with the loaded foreground masks;
+                0 value is used for background.
+        mask_depths: Whether to mask the depth maps with the loaded foreground
+            masks; 0 value is used for background.
+        image_height: The height of the returned images, masks, and depth maps;
+                aspect ratio is preserved during cropping/resizing.
+        image_width: The width of the returned images, masks, and depth maps;
+            aspect ratio is preserved during cropping/resizing.
+        box_crop: Enable cropping of the image around the bounding box inferred
+                from the foreground region of the loaded segmentation mask; masks
+                and depth maps are cropped accordingly; cameras are corrected.
+        box_crop_mask_thr: The threshold used to separate pixels into foreground
+                and background based on the foreground_probability mask; if no value
+                is greater than this threshold, the loader lowers it and repeats.
+        box_crop_context: The amount of additional padding added to each
+                dimension of the cropping bounding box, relative to box size.
+    """
+
+    path_manager: Any = None
+
+    def __init__(
+        self,
+        dataset_root,
+        load_images,
+        load_depths,
+        load_depth_masks,
+        load_masks,
+        load_point_clouds,
+        max_points,
+        mask_images,
+        mask_depths,
+        image_height,
+        image_width,
+        box_crop,
+        box_crop_mask_thr,
+        box_crop_context,
+    ):
+        self.dataset_root: str = dataset_root
+        self.load_images: bool = load_images
+        self.load_depths: bool = load_depths
+        self.load_depth_masks: bool = load_depth_masks
+        self.load_masks: bool = load_masks
+        self.load_point_clouds: bool = load_point_clouds
+        self.max_points: int = max_points
+        self.mask_images: bool = mask_images
+        self.mask_depths: bool = mask_depths
+        self.image_height: int = image_height
+        self.image_width: int = image_width
+        self.box_crop: bool = box_crop
+        self.box_crop_mask_thr: float = box_crop_mask_thr
+        self.box_crop_context: float = box_crop_context
+
+    def load(
```

Comment:
Since this method returns FrameData, it will be unclear if it modifies the object or copies it. Do we need to return it? At least please make sure to document that frame_data is modified in-place.